backend/src/server.ts

```diff
@@ -1,17 +1,102 @@
 import express from 'express';
+import WebSocket from 'ws';
+import * as fs from 'fs';
+
 import * as code from './code';
+import { handleConnection } from './connection';
 
 const app = express();
-const used: string[] = [];
+const usedGame: string[] = [];
+// used ids for both players and host
+const usedId: string[] = [];
+// first key is gameId, second key is attributes (data, host)
+const games: Map<string, Map<string, string>> = new Map();
+
 
 app.get('/', (_req, res) => {
 	res.send('Hello, world!');
 });
 
-app.get('/code', (_req, res) => {
-	res.send(code.gen(5, used));
+app.post('/games', express.json(), (req, res) => {
```

Comment:
@celdond has added `app.use(express.json())` since you forked this branch, so when we merge we can take out json() here as that is just enabled for every route now